plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcCachingConnectorSmokeTest.java

```diff
@@ -16,27 +16,79 @@
 import com.google.common.collect.ImmutableMap;
 import io.trino.testing.QueryRunner;
 import io.trino.testing.TestingConnectorBehavior;
+import io.trino.testing.sql.JdbcSqlExecutor;
+import org.testng.annotations.Test;
 
+import java.nio.file.Path;
 import java.util.Map;
+import java.util.Properties;
 
 import static io.trino.plugin.jdbc.H2QueryRunner.createH2QueryRunner;
+import static io.trino.plugin.jdbc.mapping.RuleBasedIdentifierMappingUtils.createRuleBasedIdentifierMappingFile;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class TestJdbcCachingConnectorSmokeTest
         extends BaseJdbcConnectorSmokeTest
 {
+    private JdbcSqlExecutor h2SqlExecutor;
+
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
+        Path mappingFile = createRuleBasedIdentifierMappingFile();
         Map<String, String> properties = ImmutableMap.<String, String>builder()
                 .putAll(TestingH2JdbcModule.createProperties())
                 .put("metadata.cache-ttl", "10m")
                 .put("metadata.cache-missing", "true")
+                .put("case-insensitive-name-matching", "true")
+                .put("case-insensitive-name-matching.config-file", mappingFile.toFile().getAbsolutePath())
+                .put("case-insensitive-name-matching.config-file.refresh-period", "100s")
```

Comment:
Remove these 2 since you aren't actually using any "rules" within this test class.

Reply:
Yes done - leftover from experiments.